src/tests/retry.test.ts

```diff
@@ -112,4 +112,33 @@ describe('Retry', () => {
     expect(fn).toHaveBeenCalledTimes(2);
     expect(result).toStrictEqual({ success: true });
   })
+
+  it('should back off between retries based on backoffFactor', async () => {
+    const startTime = new Date().getTime();
```

Comment:
I think we should move it closer to the `retryTest` function!

Reply:
Oops, true. I fixed it.